yarn.lock

```diff
@@ -5023,6 +5023,10 @@ safe-buffer@^5.0.1, safe-buffer@~5.0.1:
   version "5.0.1"
   resolved "https://registry.yarnpkg.com/safe-buffer/-/safe-buffer-5.0.1.tgz#d263ca54696cd8a306b5ca6551e92de57918fbe7"
 
+sdp@^1.5.0:
```

Comment:
I'm wondering why there are two new dependencies in the `yarn.lock` file. Any ideas?